README.md

```diff
@@ -16,10 +16,42 @@ For this, we use an ADR-inspired process.
 
 To make a meaningful contribution:
 
-1. Create an ADR proposal in the [decisions/log](decisions/log/) with status `In Progress`
-    1. If possible, already create the changes to the technical specification text as well (see above)
-2. Create a Pull Request with the ADR proposal and optionally the specification changes
-3. Once there is consensus on the ADR proposal, the ADR proposal is merged into the `main` branch
+## ADR Submission, Review, and Acceptance Process
```

Comment:
Thanks Prasad for updating this process, the updated version is much clearer. I shared a few comments. Also, after we discuss the backlog refinement and feedback process, this ADR process can also be updated again as it should become clearer how ADR feedback is raised vs. all other feedback (i.e. via Notion). Let's discuss!

Reply:
Thank you for your comments and suggestions, Beth! I sync with @prasadt1 on this and made the relevant changes. Happy to help in any further changes and/or improvements!

Note: the review by `trivets-rhodium` is mine and (as explained in another comment) was just a quick test.

Reply:
Nice! Thanks @raimundo-henriques!